.gitlab-ci.yml

```diff
@@ -582,6 +582,10 @@ API:
         RUNNER:
           - aws/fedora-35-x86_64
           - aws/fedora-36-x86_64
+      - IMAGE_TYPE: ["aws"]
+        RUNNER:
+          - aws/rhel-8.6-ga-aarch64
```

Comment:
Oh, just one thing, can we enable this also on rhel-9.0? I would like to have coverage on both distros. Can be done in a follow-up.

Reply:
@diaasami agreed to add rhel-9.0 in a followup PR.